[
  {
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/9739324",
    "html_url": "https://github.com/bitcoin/bitcoin/pull/1954#issuecomment-9739324",
    "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/1954",
    "id": 9739324,
    "node_id": "MDEyOklzc3VlQ29tbWVudDk3MzkzMjQ=",
    "user": {
      "login": "sipa",
      "id": 548488,
      "node_id": "MDQ6VXNlcjU0ODQ4OA==",
      "avatar_url": "https://avatars.githubusercontent.com/u/548488?u=7999a16349f0df0fb273fffa18e5a955c9d3f11c&v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/sipa",
      "html_url": "https://github.com/sipa",
      "followers_url": "https://api.github.com/users/sipa/followers",
      "following_url": "https://api.github.com/users/sipa/following{/other_user}",
      "gists_url": "https://api.github.com/users/sipa/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/sipa/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/sipa/subscriptions",
      "organizations_url": "https://api.github.com/users/sipa/orgs",
      "repos_url": "https://api.github.com/users/sipa/repos",
      "events_url": "https://api.github.com/users/sipa/events{/privacy}",
      "received_events_url": "https://api.github.com/users/sipa/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2012-10-24T13:34:31Z",
    "updated_at": "2012-10-24T13:34:31Z",
    "author_association": "MEMBER",
    "body": "If a -datadir is specified and there's a problem with it, you want to get an error, not a default.\n",
    "reactions": {
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/9739324/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/9739824",
    "html_url": "https://github.com/bitcoin/bitcoin/pull/1954#issuecomment-9739824",
    "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/1954",
    "id": 9739824,
    "node_id": "MDEyOklzc3VlQ29tbWVudDk3Mzk4MjQ=",
    "user": {
      "login": "Diapolo",
      "id": 1419649,
      "node_id": "MDQ6VXNlcjE0MTk2NDk=",
      "avatar_url": "https://avatars.githubusercontent.com/u/1419649?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/Diapolo",
      "html_url": "https://github.com/Diapolo",
      "followers_url": "https://api.github.com/users/Diapolo/followers",
      "following_url": "https://api.github.com/users/Diapolo/following{/other_user}",
      "gists_url": "https://api.github.com/users/Diapolo/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/Diapolo/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/Diapolo/subscriptions",
      "organizations_url": "https://api.github.com/users/Diapolo/orgs",
      "repos_url": "https://api.github.com/users/Diapolo/repos",
      "events_url": "https://api.github.com/users/Diapolo/events{/privacy}",
      "received_events_url": "https://api.github.com/users/Diapolo/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2012-10-24T13:47:30Z",
    "updated_at": "2012-10-24T13:52:18Z",
    "author_association": "NONE",
    "body": "Well that is indeed also true but just leads to the command-line parsing discussion and silent errors. Perhaps I missunderstood what is causing the crash ... it was bitcoind and seems to be caused by the Shutdown() call. I guess there is no need for this patch then ^^, sorry.\n",
    "reactions": {
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/9739824/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  }
]